src/plugins/azgenerator/CodeModelAzImpl.ts

```diff
@@ -1762,6 +1762,9 @@ export class CodeModelCliImpl implements CodeModelAz {
                     for (let method_param of method_param_list) {
                         let polySubParamObj = method_param.value;
                         if (polySubParamObj['polyBaseParam'] == methodParam.value) {
+                            if (!polySubParamObj.schema.extensions) {
+                                break;
```

Comment:
Is it possible to avoid using break in code? Try:
If (polySubParamObj.schema.extensions){
    logic that extensions is exists. 
}